data-prepper-plugins/key-value-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessorTests.java

```diff
@@ -315,6 +316,95 @@ void testIncludeExcludeKeysOverlapKeyValueProcessor() {
         assertThrows(IllegalArgumentException.class, () -> new KeyValueProcessor(pluginMetrics, mockConfig));
     }
 
+    @Test
+    void testDefaultKeysKeyValueProcessor() {
+        final Map<String, Object> defaultMap = Map.of("dKey", "dValue");
+        when(mockConfig.getDefaultKeys()).thenReturn(defaultMap);
+        keyValueProcessor = new KeyValueProcessor(pluginMetrics, mockConfig);
+
+        final Record<Event> record = getMessage("key1=value1");
+        final List<Record<Event>> editedRecords = (List<Record<Event>>) keyValueProcessor.doExecute(Collections.singletonList(record));
+        final LinkedHashMap<String, Object> parsed_message = getLinkedHashMap(editedRecords);
```

Comment:
nit: Using underscores in variable names is not a Java pattern. Prefer `parsedMessage` over `parsed_message`.

Reply:
We should be consistent with the camel variable style. same below

Reply:
Yes, this is still using underscore.